Source/WebKit/SourcesCocoa.txt

```diff
@@ -80,9 +80,9 @@ GPUProcess/webrtc/RemoteAudioMediaStreamTrackRendererInternalUnitManager.cpp
 Platform/classifier/cocoa/ResourceLoadStatisticsClassifierCocoa.cpp
 Platform/classifier/ResourceLoadStatisticsClassifier.cpp
 
-Platform/cocoa/NetworkConnectionIntegrityHelpers.mm
 Platform/cocoa/PaymentAuthorizationPresenter.mm
 Platform/cocoa/PaymentAuthorizationViewController.mm
+Platform/cocoa/WebPrivacyHelpers.mm
```

Comment:
Nit: Sort order would put this below `Platform/cocoa/WebKitAdditions.mm @no-unify`.

Unless there's a reason it needs to be before?

Reply:
Good catch — fixed!